linker/shared/src/main/scala/org/scalajs/linker/frontend/optimizer/OptimizerCore.scala

```diff
@@ -5096,33 +5100,6 @@ private[optimizer] abstract class OptimizerCore(
     }
   }
 
-  private def foldJSSelect(qualifier: Tree, item: Tree)(
-      implicit pos: Position): Tree = {
-    // !!! Must be in sync with scala.scalajs.runtime.LinkingInfo
```

Comment:
It's really nice that we can get rid of this :)

Reply:
That said, this change belongs to the first commit :)

Reply:
Right, moved to the first commit :+1: